src/webapp/controllers/datasetSelectorController.js

```diff
@@ -21,7 +21,7 @@ function controller($scope, $rootScope, DataSetsUID, Locales) {
 			for (att in ds.attributeValues) {
```

Comment:
That's weird, this seems to be coming from the old code so it must have been working, but on my test (installed as app), I get `ReferenceError: att is not defined`. A usual `for (att in ds.attributeValues)` fixes it.